tpl/dash/dashboard.tpl.php

```diff
@@ -30,6 +30,8 @@
 $ucss_count = count($this->load_queue('ucss'));
 $placeholder_queue_count = count($this->load_queue('lqip'));
 $vpi_queue_count = count($this->load_queue('vpi'));
+
+$can_page_load_time = defined('LITESPEED_SERVER_TYPE') && LITESPEED_SERVER_TYPE != 'NONE' ? true : false;
```

Comment:
This is getting dangerous.
I promise I will never tell anyone about it!
```suggestion
$can_page_load_time = defined('LITESPEED_SERVER_TYPE') && LITESPEED_SERVER_TYPE !== 'NONE';
```

Reply:
@szepeviktor what is hgetting dangerous? didn't get that
Also made the changes.

Reply:
@szepeviktor Can you explain more on this point? TIA

Reply:
@szepeviktor because: you can define LITESPEED_SERVER_TYPE ?

Reply:
> defined(string $constant_name): bool

https://www.php.net/manual/en/function.defined.php